src/components/loader-star/loader-star.test.tsx

```diff
@@ -49,7 +49,7 @@ test("component should have a visually hidden label for assistive technologies b
 });
 
 test("renders a custom visually hidden label for assistive technologies to use, when loaderStarLabel prop is provided", () => {
-  mockUseMediaQuery.mockReturnValueOnce(true);
+  mockUseMediaQuery.mockReturnValueOnce(false);
```

Comment:
comment: the hidden label will only render when motion is allowed so this should be `true`